Doc/extending/newtypes_tutorial.rst

```diff
@@ -116,12 +115,14 @@ field mentioned above. ::
    .tp_name = "noddy.Noddy",
 
 The name of our type.  This will appear in the default textual representation of
-our objects and in some error messages, for example::
+our objects and in some error messages, for example:
 
-   >>> "" + noddy.new_noddy()
+.. code-block:: python
```

Comment:
"[pycon](http://pygments.org/docs/lexers/#pygments.lexers.python.PythonConsoleLexer)" may be better option.

Reply:
I didn't know about that.

Reply:
Ok, it doesn't seem to make a difference compared to "python".